build_daemon/lib/src/server.dart

```diff
@@ -29,6 +30,7 @@ import 'managers/build_target_manager.dart';
 /// Handles notifying clients of logs and results for registered build targets.
 /// Note the server will only notify clients of pertinent events.
 class Server {
+  static final _logger = Logger('BuildDaemon Server');
```

Comment:
I don't think we want to use a `Logger` here but instead the `logHandler` that is part of this package. The reason being that the daemon process doesn't log to a user visible output while the `logHandler` pipes logs to the clients for display.